pkg/util/azureclient/authz/remotepdp/const.go

```diff
@@ -0,0 +1,17 @@
+package remotepdp
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+const (
+	modulename = "aro-pdpclient"
+	// version is the semantic version of this module
+	version = "0.0.1" //nolint
```

Comment:
why do we have nolint here?

Reply:
I got this from the applens code.  I can take it out and see what happens.

Reply:
Removed `nolint` in the latest commit.